cli/fathom_web/cert/cert.pem

```diff
@@ -0,0 +1,50 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
I feel like we might as well move the single file out of the dir and let it be top level. No?

Reply:
I do think having a directory named `cert` is a bit much. While we just have this single file, I think letting it be top level is fine. If we start having more data files, perhaps we'll want a `data` directory.